cognite_toolkit/_cdf_tk/loaders/_resource_loaders/classic_loaders.py

```diff
@@ -59,18 +59,25 @@ def dump_id(cls, id: str) -> dict[str, Any]:
         return {"externalId": id}
 
     @classmethod
-    def get_required_capability(cls, items: AssetWriteList | None) -> Capability | list[Capability]:
+    def get_required_capability(cls, items: AssetWriteList | None, read_only: bool) -> Capability | list[Capability]:
         if not items and items is not None:
             return []
         scope: capabilities.AssetsAcl.Scope.All | capabilities.AssetsAcl.Scope.DataSet = (  # type: ignore[valid-type]
             capabilities.AssetsAcl.Scope.All()
         )
+
+        actions = (
+            [capabilities.AssetsAcl.Action.Read]
+            if read_only
+            else [capabilities.AssetsAcl.Action.Read, capabilities.AssetsAcl.Action.Write]
+        )
```

Comment:
Just an alternative way to write the same (I do not expect you to rewrite, it is just an FYI)
```suggestion
        actions = [capabilities.AssetsAcl.Action.Read]
        if not read_only:
            actions.append(capabilities.AssetsAcl.Action.Write)
```